src/cli.zig

```diff
@@ -1988,27 +2022,31 @@ pub const Command = struct {
                 },
                 Command.Tag.CreateCommand => {
                     const intro_text =
-                        \\<b>Usage<r>:
-                        \\  <b><green>bun create<r> <blue>\<template\><r> <cyan>[...flags]<r> <blue>[dest]<r>
-                        \\  <b><green>bun create<r> <blue>\<username/repo\><r> <cyan>[...flags]<r> <blue>[dest]<r>
-                        \\
-                        \\<b>Environment variables:<r>
-                        \\  <cyan>GITHUB_ACCESS_TOKEN<r>      <d>Supply a token to download code from GitHub with a higher rate limit<r>
-                        \\  <cyan>GITHUB_API_DOMAIN<r>        <d>Configure custom/enterprise GitHub domain. Default "api.github.com".<r>
-                        \\  <cyan>NPM_CLIENT<r>               <d>Absolute path to the npm client executable<r>
+                        \\<b>Usage<r>: <b><green>bun create<r> <cyan>\<template\><r> [\<destination\>]
+                        \\<b>Alias: <b>bun c<r>
+                        \\Create a new bun project using a create-\<template\> npm package, GitHub repo, or a local template
                     ;
 
                     const outro_text =
-                        \\If given a GitHub repository name, Bun will download it and use it as a template,
-                        \\otherwise it will run <b><magenta>bunx create-\<template\><r> with the given arguments.
+                        \\<b>Examples:<r>
+                        \\  <b>From npm :<r>
+                        \\  <b><green>bun create remix<r>
+                        \\
+                        \\  <b>From GitHub :<r>
+                        \\  <b><green>bun create github.com/\<user\>/\<repo\><r>
```

Comment:
```suggestion
                        \\  <d>From GitHub<r>
                        \\  <b><green>bun create \<user\>/\<repo\><r>
```
maybe we can pick an actual example, idk

Reply:
I think we can keep the basic example of the doc as you put in your suggestion. The link to the full documentation is in the bottom of the man command :)